### ISSUE TITLE
question for version

### ISSUE THREAD

**Issue by jind11** [FrankWork/acnn#4]
I noticed that in the log file, the version "baseline+attentive pooling" can get the result: 05-10 21:12 Epoch: 21 Train: 94.81% Test: 75.19%. What are the model configurations in details for this result? If possible, could you send me the model file for this result? My email is jindi15@mit.edu. I have tried my best but cannot reach this performance. Thank you so much!

**Comment by FrankWork** [comment]
@jind11 the training is very fast even on cpu, so you can train you own model:

```bash
$ git pull git@github.com:FrankWork/acnn.git
$ git pull origin baseap
$ git checkout baseap
$ python main.py
```
I rerun the program, and got the following results:
```
12-08 14:22 Epoch: 16 Train: 74.76% Test: 73.93%
12-08 14:22 Epoch: 17 Train: 74.95% Test: 74.59%
12-08 14:22 Epoch: 18 Train: 75.59% Test: 74.56%
12-08 14:23 Epoch: 19 Train: 76.80% Test: 74.22%
12-08 14:23 Epoch: 20 Train: 77.56% Test: 73.93%
12-08 14:23 Epoch: 21 Train: 78.33% Test: 74.78%
12-08 14:23 Epoch: 22 Train: 78.45% Test: 75.04%
12-08 14:24 Epoch: 23 Train: 79.99% Test: 75.30%
12-08 14:24 Epoch: 24 Train: 80.29% Test: 75.44%
12-08 14:24 Epoch: 25 Train: 79.96% Test: 75.15%
```

**Comment by FrankWork** [comment]
If you can reach the performance in the paper, please let me know.

**Comment by jind11** [comment]
I have one question: where do you get the pre-trained embedding file? Is it trained on English Wikipedia? Thanks!

**Comment by FrankWork** [comment]
@jind11 trained on Wikipedia

https://github.com/Hironsan/awesome-embedding-models

https://ronan.collobert.com/senna/senna-v3.0.tgz

**Comment by jind11** [comment]
I see. According to my experiment on different embedding sources, the pre-trained embeddings has influence on the performance. I am gonna train word2vec 300 and 400 dimension on English Wikipedia myself.

**Comment by zzzyj** [comment]
The official F1 score of the SemEval Task regards the task as a (9+1)-way classification with directionality taken into account, and it is much higher than the accuracy.
